contracts/CoveragePool.sol

```diff
@@ -140,4 +161,26 @@ contract CoveragePool is Ownable {
 
         assetPool.claim(recipient, amountToSeize);
     }
+
+    function getRemainingRiskManagerApprovalTime(address riskManager)
+        external
+        view
+        returns (uint256)
+    {
+        require(
+            riskManagerApprovalTimestamps[riskManager] > 0,
+            "Risk manager approval not initiated"
+        );
+        uint256 elapsed =
+            /* solhint-disable-next-line not-rely-on-time */
```

Comment:
I am confused. Linter is complaining but there is no public view function here. 🤔